src/utils/event.js

```diff
@@ -39,4 +35,16 @@ function getTotalEventPrice(price, offers) {
   }
 }
 
-export { getRandomArrayElement, getFormattedEventDate, getEventDurationTime, getTotalEventPrice };
+function isFutureEvent(startDate) {
+  return startDate && dayjs().isAfter(startDate, 'D');
+}
+
+function isPresentEvent(startDate, endDate) {
+  return startDate && endDate && (dayjs().isBefore(startDate, 'D') || dayjs().isSame(startDate, 'D')) && (dayjs().isAfter(endDate, 'D') || dayjs().isSame(endDate, 'D'));
+}
+
+function isPastEvent(endDate) {
+  return endDate && dayjs().isBefore(endDate, 'D');
+}
```

Comment:
У нас есть критерий про избыточные проверки. Поэтому наличие аргументов можно не проверять

```suggestion
function isFutureEvent(startDate) {
  return dayjs().isAfter(startDate, 'D');
}

function isPresentEvent(startDate, endDate) {
  return (dayjs().isBefore(startDate, 'D') || dayjs().isSame(startDate, 'D')) && (dayjs().isAfter(endDate, 'D') || dayjs().isSame(endDate, 'D'));
}

function isPastEvent(endDate) {
  return dayjs().isBefore(endDate, 'D');
}
```